website/client/src/components/userMenu/profile.vue

```diff
@@ -838,6 +839,11 @@ export default {
   mounted () {
     this.loadUser();
     this.selectPage(this.startingPage);
+    this.$root.$on('restoreTitle', () => {
```

Comment:
to make sure we don't keep this event listener even after the component is not being used it should be removed before the component is destroyed with the following hook(you can place it right after the `mounted` hook

```
  beforeDestroy () {
    this.$root.$off('habitica: restoreTitle');
  },
```